docs/data/joy/components/alert/alert.md

```diff
@@ -27,28 +36,57 @@ export default function MyApp() {
 }
 ```
 
+### Basics
+
+The Alert component wraps around its content:
+
+```jsx
+<Alert>This is a time-sensitive message.</Alert>
+```
+
+### Anatomy
+
+The Alert component is composed of a single root `<div>` element with its `role` set to `alert`:
+
+```html
+<div role="alert" class="JoyAlert-root">
+  This is how an Alert renders in the DOM.
+</div>
+```
+
+### Overriding the root slot
+
+Use the `component` prop to override the root slot with a custom element.
+For example, the following code snippet replaces the default `<div>` with a `<span>`:
+
+```jsx
+<Alert component="span" />
+```
+
+## Customization
```

Comment:
```suggestion
## Props
```
**Customization** is quite broad and for Joy UI, maybe it is lean toward styling.

Using **Props** could be more direct. What do you think?

Reply:
I'm with Sam on this one because I think that almost all the API of the components are props.

> so we'll have consistency across the products

Then I think we should first check it it's compatible with Material UI because the Joy UI is closer to it than MUI Base.

Reply:
I think the structure that I'm using for the updated Joy docs will work well enough for Material UI going forward. It's very close to what I created for the Base docs but with some minor adjustments to better suit the product, and I think the same thing will happen when I eventually move on from Joy to Material UI.

Here's a related question: Many of these component pages that are in both Joy & Material UI docs contain shared/copied content. It would be kind of silly for me to come up with new ways to paraphrase these descriptions for each product, so it makes sense that some of the content would be repeated. With that in mind, maybe I should revise the Joy and Material UI pages at the same time (wherever they share components)? It would take a bit longer to complete the project on the Joy UI side, but it would get the ball rolling for Material UI component page revisions. And then we could confirm that the updated structure works for both products, or else adjust as needed.

Reply:
Just for fun 😅 I revised the Material UI Alert page to see if it makes sense to work on these pages together. #34892